lib/semantics/check-select-stmt.cc

```diff
@@ -0,0 +1,121 @@
+//==check-select-stmt.cc - Checker for select-rank
```

Comment:
The first and last lines of this license block are wrong.  Please see other files and conform to their usage.